### ISSUE TITLE
Including event queue stats in live replication output

### ISSUE THREAD

**Issue by samongyr-sq** [redis-developer/riot#115]
When using the `replicate` subcommand with `--mode live` (or `--mode liveonly`), there's no way to determine when the internal queue is full from the current output 

Including information about the the size of the event queue relative to its capacity would make it easier to determine when `--event-queue` and/or `--read-queue` should be configured to prevent updates from being dropped

### Version Info

riot v3.0.4 

```
$ riot <source> replicate <target> --mode liveonly --progress ascii 

Listening  ? % [                    =                          ] 20/? (0:00:20 / ?) 1.0/s 
```

**Comment by jruaux** [comment]
🎉 This issue has been resolved in `v3.1.1` ([Release Notes](https://github.com/redis-developer/riot/releases/tag/v3.1.1))

**Comment by pamelayin** [comment]
@jruaux This seems to be fixed for above command at `v3.1.1` but it doesn't seem to work when the source is elasticache and target is redis on-prem with `--type ds`.
```
/riot-3.1.1/bin/riot <elasticache source> replicate <redis target> --mode live --type ds
Listening  ? % [=                                                 ] 0/? (0:00:00 / ?) ?/s
Scanning 100% [=========================] 4000/4000 (0:00:06 / 0:00:00) 666.7/s
```
